pmm-app/src/pmm-check/components/FailedChecksTab/FailedChecksTab.utils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { saveShowSilencedValue, loadShowSilencedValue } from './FailedChecksTab.utils';
+import { SHOW_SILENCED_VALUE_KEY, SHOW_SILENCED_DEFAULT } from './FailedChecksTab.constants';
+
+let getItemSpy: jest.SpyInstance;
+let setItemSpy: jest.SpyInstance;
+
+describe('FailedChecksTab::utils', () => {
+  beforeEach(() => {
+    getItemSpy = jest.spyOn(Storage.prototype, 'getItem').mockImplementation(() => 'true');
+    setItemSpy = jest.spyOn(Storage.prototype, 'setItem');
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('saveShowSilencedValue calls localStorage.setItem', () => {
+    saveShowSilencedValue(true);
+
+    expect(setItemSpy).toBeCalledTimes(1);
+    expect(setItemSpy).toBeCalledWith(SHOW_SILENCED_VALUE_KEY, 'true');
+  });
+
+  test('loadShowSilencedValue calls localStorage.getItem', () => {
+    expect(getItemSpy).toBeCalledTimes(1);
+    expect(loadShowSilencedValue()).toEqual(true);
+  });
+
+  test('loadShowSilencedValue the default value if localStorage is not available', () => {
+    jest.spyOn(Storage.prototype, 'getItem').mockImplementation(() => { throw Error('test error'); });
```

Comment:
Since we are testing error throwing, should we "silence" the console during tests?
I've done that [here](https://github.com/percona-platform/grafana/blob/0852d323bc2fb6608065859cbc858e5ab2541f18/public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplate.test.tsx#L16) because it was too hard to follow the test output.

Reply:
Done!